core/src/main/java/apoc/create/Create.java

```diff
@@ -220,6 +221,33 @@ public Relationship vRelationshipFunction(@Name("from") Node from, @Name("relTyp
         return Stream.of(new VirtualPathResult(from, rel, to));
     }
 
+    @Procedure
+    @Description("apoc.create.vPath")
+    public Stream<PathResult> vPath(@Name("path") Path path) {
+        return Stream.of(createVPath(path));
+    }
+
+    @Procedure
+    @Description("apoc.create.vPaths")
+    public Stream<PathResult> vPaths(@Name("paths") List<Path> paths) {
+        return paths.stream().map(this::createVPath);
+    }
+
+    private PathResult createVPath(Path path) {
+        final Iterable<Relationship> relationships = path.relationships();
+        final Node first = path.startNode();
+        PathImpl.Builder builder = new PathImpl.Builder(new VirtualNode(first, Iterables.asList(first.getPropertyKeys())));
+        for (Relationship rel : relationships) {
+            Node startNode = rel.getStartNode();
+            Node endNode = rel.getEndNode();
+            Node start = new VirtualNode(startNode, Iterables.asList(startNode.getPropertyKeys()));
```

Comment:
per favore crea un metodo statico dentro VirtualNode che implementa quella business logic
```suggestion
            Node start = VirtualNode.from(startNode);
```